lib/cryptoexchange/exchanges/airswap/services/market.rb

```diff
@@ -0,0 +1,49 @@
+module Cryptoexchange::Exchanges
+  module Airswap
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all output
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Airswap::Market::API_URL}/ticker"
+        end
+
+        def adapt_all(output)
+          output.map do |pair, output|
+            base, target = pair.split('_')
```

Comment:
it looks like airswap returns pairs like ETH_DAI, ETH_FUN

ETH is the target, DAI is the base
need to reverse the assignment